Userland/Libraries/LibWeb/CSS/Parser/Tokenizer.cpp

```diff
@@ -427,11 +426,11 @@ Token Tokenizer::consume_an_ident_like_token()
     // It returns an <ident-token>, <function-token>, <url-token>, or <bad-url-token>.
 
     // Consume an ident sequence, and let string be the result.
-    auto string = consume_an_ident_sequence();
+    auto string = consume_an_ident_sequence().release_value_but_fixme_should_propagate_errors();
 
     // If string’s value is an ASCII case-insensitive match for "url", and the next input code
     // point is U+0028 LEFT PARENTHESIS ((), consume it.
-    if (string.equals_ignoring_case("url"sv) && is_left_paren(peek_code_point())) {
+    if (string.bytes_as_string_view().equals_ignoring_case("url"sv) && is_left_paren(peek_code_point())) {
```

Comment:
Hmm, this says to ASCII compare, but I'm not sure how I feel about relying on "String is Unicode, StringView is ASCII". Ideally `StringView` will also be Unicode aware once the dust settles, so cases like this might be better served by an explicit ASCII comparator function (that VERIFYs all code points are indeed ASCII as it goes).

Reply:
Thanks for giving me the kick to actually implement this properly! From the spec, it doesn't require that the whole string is ASCII, it just defines the ASCII lowercase of a string to be one where the A-Z codepoints are made lowercase.